docs/reference/data-streams/downsampling-manual.asciidoc

```diff
@@ -0,0 +1,450 @@
+                                            "bytes": {
+                                                "type": "long",
+                                                "time_series_metric": "gauge"
+                                            },
+                                            "limit": {
+                                                "properties": {
+                                                    "pct": {
+                                                        "type": "float"
+                                                    }
+                                                }
+                                            },
+                                            "node": {
+                                                "properties": {
+                                                    "pct": {
+                                                        "type": "float"
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    },
+                                    "workingset": {
+                                        "properties": {
+                                            "bytes": {
+                                                "type": "long",
+                                                "time_series_metric": "gauge"
+                                            }
+                                        }
+                                    }
+                                }
+                            },
+                            "name": {
+                                "type": "keyword"
+                            },
+                            "start_time": {
+                                "type": "date"
+                            }
+                        }
+                    },
+                    "host": {
+                        "type": "keyword",
+                        "time_series_dimension": true
+                    },
+                    "namespace": {
+                        "type": "keyword",
+                        "time_series_dimension": true
+                    },
+                    "node": {
+                        "type": "keyword",
+                        "time_series_dimension": true
+                    },
+                    "pod": {
+                        "type": "keyword",
+                        "time_series_dimension": true
+                    }
+                }
+            }
+        }
+    }
+}
+
+----
+// TEST
+
+[discrete]
+[[downsampling-manual-ingest-data]]
+==== Ingest time series data
+
+In a terminal window with {es} running, run the following curl command to load the documents from the downloaded sample data file:
+
+//[source,console]
+//----
+```
+curl -s -H "Content-Type: application/json" \
+   -XPOST http://localhost:9200/sample-01/_bulk?pretty \
```

Comment:
```suggestion
   -XPOST http://<elasticesearch-node>:9200/sample-01/_bulk?pretty \
```

Reply:
Fixed in [507419d](https://github.com/elastic/elasticsearch/pull/88571/commits/507419dcc6e4498ad50f37505d150e5de4db8a77)